titan/params/prep.yml

```diff
@@ -21,7 +21,6 @@ prep:
     default: 0
     description: Timestep at which prep should start in the model
     type: int
```

Comment:
with the change to `target_as_prob`, was the demographic corollary also updated to `target?

Reply:
Good catch; looks like we were never consistent between `prep.yml` and `demographics.yml`

Reply:
also, is the prep.target description right?